resources/public/api/conf/3.0/schemas/retrieveBiss/def1/response.json

```diff
@@ -52,6 +52,11 @@
           "description": "The taxable net profit of the income source. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
           "type": "number",
           "example": "0.00"
+        },
+        "adjusted": {
+          "description": "Profit including disallowable expenses and deducting non‑taxable income, before claiming capital allowances or making any further tax adjustments. The value must be between 0 and 99999999999.99 up to 2 decimal places.",
```

Comment:
`This field will only be returned for tax year 2025-26 and onwards.` This needs to be specified here too as we don't have a separate schema for the tax year.